### PR TITLE
Update regex add timeframe

### PR DESCRIPTION
**Pull request by jaap**
Combined pull request to address #3 and update the regex so the use of ' in city names will work as well.

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -36,12 +36,16 @@ public function getDaily(\DateTime $start, \DateTime $end, array $stations, arra
         return (new ResponseParser())->parse($this->httpClient->sendRequest($request)->getBody()->getContents());
     }
 
-    public function getHourly(\DateTime $start, \DateTime $end, array $stations, array $vars) : array
+    public function getHourly(\DateTime $start, \DateTime $end, $timeframe_start, $timeframe_end, array $stations, array $vars) : array
```
Comment (ricbra) [inline src/Client.php]: Could you change the new variables to $camelCase?

File: test/ClientTest.php [hunk 2]
```diff
@@ -7,6 +7,7 @@
 use Http\Discovery\MessageFactoryDiscovery;
 use Http\Mock\Client as MockClient;
 use Ricbra\Knmi\Client;
+use PHPUnit\Framework\TestCase;
```
Comment (ricbra) [inline test/ClientTest.php]: This can also be removed now

File: test/ResponseParserTest.php [hunk 3]
```diff
@@ -3,6 +3,7 @@
 namespace Test\Ricbra\Knmi;
 
 use Ricbra\Knmi\ResponseParser;
+use PHPUnit\Framework\TestCase;
```
Comment (ricbra) [inline test/ResponseParserTest.php]: Can also be removed

**Comment by ricbra** [comment]
A couple of small issues and we're good to go! 👍

File: test/ClientTest.php (hunk 2 above)
Comment (jaap) [reply test/ClientTest.php]: Done

File: src/Client.php (hunk 1 above)
Comment (jaap) [reply src/Client.php]: Check!

**Comment by jaap** [comment]
Welcome. Thanks for the merge!

**Pull request merged**